src/Akka.Hosting/Logging/LoggerFactoryLogger.cs

```diff
@@ -26,12 +26,18 @@ public class LoggerFactoryLogger: ActorBase, IRequiresMessageQueue<ILoggerMessag
 
         private readonly ConcurrentDictionary<Type, ILogger> _loggerCache = new ConcurrentDictionary<Type, ILogger>();
```

Comment:
This does not need to be concurrent, still.

Reply:
Done